docs/GetStarted/getting_started_linux_cmake.md

```diff
@@ -71,13 +72,13 @@ $ git submodule update --init
 Configure:
 
 ```shell
-$ cmake -G Ninja -B build/ -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ .
```

Comment:
As commented in the discussion, this is *necessary on most distributions*!

Reply:
Fixed!